### PR TITLE
Adds ability to handle more than 2 saige inputs

### PR DESCRIPTION
**Pull request by mmeier93**
## This PR does the following:

- It adds the ability for the pipeline to handle any number of `SAIGE` outputs (outputs of `lifebit-ai/gel-gwas`).
- It removes remaining nf-core template related files that are not needed.
- It adds the first `lifebit-ai/metagwas` `Dockerfile` and `environment.yml` files.

## Details:

- Modified the original channel - now called `all_input_studies_ch`, and takes in a `*csv` file that lists the `SAIGE` inputs. The channel reads the file and then stages that study files.

- Modified the `run_metal` process: added `Bash` commands to dynamically process the number of input files. Uses a `for loop` along with `echo`. This ends with a `process_commands` commands that is then added inside the `METAL` script.

- Added a `Dockerfile` and `environment.yml` (pipeline was previously using the `lifebitai/metal` image. Added `METAL` and basic `R` packages (as an anticipation for future features of the pipeline - namely the ability to handle different inputs that SAIGE, which will mean wrangling with `R`).

- Removed unused nf-core template related files.

## Comments:

The input `list-summary-statistics.csv` used for testing is present in `testdata` and in my personal S3 bucket (`testdata-magda`). The test on CloudOS is using the file in S3. This file lists s3 links to the study files, also in my own S3 bucket.

All these files will eventually need to be moved to the common S3 bucket.

## Test on CloudOS:

https://cloudos.lifebit.ai/public/jobs/5fd784f8aec0ef0112b7435e

Screenshot 2020-12-14 at 15 43 46

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -8,15 +8,14 @@
...
+process.container = 'quay.io/lifebitai/metagwas:latest'
 
 // Global default params, used in configs
 params {
 
   // 1 - Workflow flags
 
-  study_1 = false
-  study_2 = false
+  studies = "https://testdata-magda.s3-eu-west-1.amazonaws.com/metagwas/list-summary-statistics.csv"
```
Comment (cgpu) [inline nextflow.config:18]:
```suggestion
  studies = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/metagwas/list-summary-statistics.csv"
```

File: conf/basic_test.config [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ * nextflow run main.nf -profile basic_test
+ */
+
+
+docker.enabled = true
+
+params  {
+    studies = "https://testdata-magda.s3-eu-west-1.amazonaws.com/metagwas/list-summary-statistics.csv"
```
Comment (cgpu) [inline conf/basic_test.config:14]:
```Suggestion
    studies = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/metagwas/list-summary-statistics.csv"
```

**Review by cgpu — APPROVED** [review]
Good to go! 👍

**Comment by mmeier93** [comment]
> Good to go! 👍

Thanks 👍

**Pull request merged**